controllers/glance_controller.go

```diff
@@ -670,7 +670,7 @@ func (r *GlanceReconciler) reconcileNormal(ctx context.Context, instance *glance
 	// NOTE: a CronJob is created at this stage to purge all soft deleted records.
 	// This command should be executed periodically to avoid glance database becomes
 	// bigger by getting filled by soft-deleted records.
-	cronjobDef := glance.CronJob(instance, serviceLabels, serviceAnnotations)
+	cronjobDef := glance.CronJob(instance, serviceLabels, serviceAnnotations, glance.DBPurge)
```

Comment:
we still don't support cache (additional work is needed on that front), so we force `glance.DBPurge` here because is the only created cronJob.
Logic to build multiple cronJobs (if cache is enabled) will be a follow up patch.